src/SVTBarrelTracker_geo.cpp

```diff
@@ -0,0 +1,324 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Whitney Armstrong, Jonathan Witte, Shujie Li
+
+/** Curved Silicon Vertex Tracker Barrel with RSU.
+ *
+ * - Designed to process "vertex_barrel_curved.xml"
+ *
+ * - Derived from "BarrelTrackerWithFrame_geo.cpp".
+ * - Build-in RSU structure with 12 tiles and inactive areas
```

Comment:
Number of tiles is likely configurable parameter, not inherent to the geometry plugin.